plugins/modules/aci_fabric_span_dst_group.py

```diff
@@ -0,0 +1,359 @@
+        type: str
+        choices: [ version_1, version_2 ]
+      version_enforced:
+        description:
+        - Enforce SPAN version.
+        type: bool
+      source_ip:
+        description:
+        - The source IP address or prefix.
+        type: str
+        required: true
+      destination_ip:
+        description:
+        - The destination IP address.
+        type: str
+        required: true
+      flow_id:
+        description:
+        - The flow ID of the SPAN packet.
+        - The APIC defaults to C(1) when unset during creation.
+        type: int
+      ttl:
+        description:
+        - The time to live of the span session packets.
+        - The APIC defaults to C(64) when unset during creation.
+        type: int
+      mtu:
+        description:
+        - The MTU truncation size for the packets.
+        - The APIC defaults to C(1518) when unset during creation.
+        type: int
+      dscp:
+        description:
+        - The DSCP value for sending the monitored packets using ERSPAN.
+        - The APIC defaults to C(unspecified) when unset during creation.
+        type: str
+        choices: [ CS0, CS1, CS2, CS3, CS4, CS5, CS6, CS7, EF, VA, AF11, AF12, AF13, AF21, AF22, AF23, AF31, AF32, AF33, AF41, AF42, AF43, unspecified ]
+  state:
+    description:
+    - Use C(present) or C(absent) for adding or removing.
+    - Use C(query) for listing an object or multiple objects.
+    type: str
+    choices: [ absent, present, query ]
+    default: present
+  name_alias:
+    description:
+    - The alias for the current object. This relates to the nameAlias field in ACI.
+    type: str
+extends_documentation_fragment:
+- cisco.aci.aci
+- cisco.aci.annotation
+- cisco.aci.owner
+
+seealso:
+- name: APIC Management Information Model reference
+  description: More information about the internal APIC class B(span:DestGrp).
+  link: https://developer.cisco.com/docs/apic-mim-ref/
+author:
+- Akini Ross (@akinross)
+"""
+
+EXAMPLES = r"""
+- name: Add a Fabric SPAN destination group
+  cisco.aci.aci_fabric_span_dst_group:
+    host: apic
+    username: admin
+    password: SomeSecretPassword
+    destination_group: group1
+    description: Test span
+    destination_epg:
+      tenant: Test1
+      ap: ap1
+      epg: ep1
+      span_version: version_1
+      version_enforced: false
+      destination_ip: 10.0.0.1
+      source_ip: 10.0.2.1
+      ttl: 2
+      mtu: 1500
+      flow_id: 1
+      dscp: CS1
+    state: present
+  delegate_to: localhost
+
+- name: Remove a Fabric SPAN destination group
+  cisco.aci.aci_fabric_span_dst_group:
+    host: apic
+    username: admin
+    password: SomeSecretPassword
+    destination_group: group1
+    state: absent
+  delegate_to: localhost
+
+- name: Query a Fabric SPAN destination group
+  cisco.aci.aci_fabric_span_dst_group:
+    host: apic
+    username: admin
+    password: SomeSecretPassword
+    destination_group: group1
+    state: query
+  delegate_to: localhost
+
```

Comment:
Other modules in the collection usually have -> register: query_result in the query examples

Reply:
ok